modules/vidazooBidAdapter.js

```diff
@@ -19,25 +20,32 @@ function isBidRequestValid(bid) {
   return !!(params.cId && params.pId);
 }
 
-function buildRequest(bid, topWindowUrl, size, bidderRequest) {
-  const {params, bidId} = bid;
-  const {bidFloor, cId, pId, ext} = params;
-  // Prebid's util function returns AppNexus style sizes (i.e. 300x250)
-  const [width, height] = size.split('x');
-
-  const dto = {
-    method: 'GET',
-    url: `${URL}/prebid/${cId}`,
-    data: {
-      url: encodeURIComponent(topWindowUrl),
-      cb: Date.now(),
-      bidFloor: bidFloor,
-      bidId: bidId,
-      publisherId: pId,
-      consent: bidderRequest.gdprConsent && bidderRequest.gdprConsent.consentString,
-      width,
-      height
+function buildRequest(bid, topWindowUrl, sizes, bidderRequest) {
+  const { params, bidId } = bid;
+  const { bidFloor, cId, pId, ext } = params;
+  let data = {
+    url: encodeURIComponent(topWindowUrl),
+    cb: Date.now(),
+    bidFloor: bidFloor,
+    bidId: bidId,
+    publisherId: pId,
+    sizes: sizes,
+  };
+  if (bidderRequest.gdprConsent) {
+    if (bidderRequest.gdprConsent.consentString) {
+      data.gdprConsent = bidderRequest.gdprConsent.consentString;
     }
+    if (bidderRequest.gdprConsent.gdprApplies !== undefined) {
+      data.gdpr = bidderRequest.gdprConsent.gdprApplies ? 1 : 0;
+    }
+  }
+  if (bidderRequest.uspConsent) {
+    data.usPrivacy = bidderRequest.uspConsent
+  }
```

Comment:
This PR is related to 'support multi sizes', you shouldn't be adding usp support to this PR.  
I suggest you remove it from this PR, and open another PR with changes regarding USP.  

In addition, you may also want to submit a docs PR to update your bidders file (in this [folder](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders)) to show you support USP. You just need to add `usp_supported: true` to the top of your file (near the other variables).